DarkFlorist/lunaria
# Ipfs dockerize

Resolve #218 

#### Notable Changes
- use `debian` as base image
- installed kubo by copying `ipfs/kubo` binary

#### Testing

- build Lunaria locally with `docker build -t lunaria .`
- create a container with the built docker image
> $ docker container run --rm lunaria
added bafkreickot2jtfysomtkxw3wy2zdysmxcj2roshp33oebmmb4gpomqwc7e export/css/index.css
added bafkreibi4iqgpwoyncp2252fn2ynvikcdjbper2xsu5jbz4nqkyxjbe2p4 export/img/0x07865c6e87b9f70255377e024ace6630c1eaa37f.svg
...
added bafybeiajyjn6zx7rht6xxv37difjvl4oahjkmsfvikhabkobuzkq2utofe export

- check that ipfs hash matched the generated hash (from ipfs_hash.txt) during build
> $ docker run --entrypoint bin/sh lunaria -c "cat ipfs_hash.txt"
bafybeiajyjn6zx7rht6xxv37difjvl4oahjkmsfvikhabkobuzkq2utofe
